examples/createLayer_custom_UI_functions.html

```diff
@@ -0,0 +1,298 @@
+         
+        },
+        {
+          type: "http",
+          urlTemplate: "http://{s}.basemaps.cartocdn.com/dark_only_labels/{z}/{x}/{y}.png",
+          subdomains: [ "a", "b", "c" ]
+        }
+      ]
+    })
+  .addTo(map) // add cartodb layer and basemap to map object
+  .done(function(layer) {
+
+    /*Once that the basemap and cartodb layer are added....*/
+      
+        /* Enable mouse events with CartoDB layers */
+
+        // 1- Set CartoDB layer interaction
+        layer.setInteraction(true);
+
+        // 2- Custom mouseover event on CartoDB layers
+        layer.on('featureOver',function(e,latlng,pos,data){
+          lat = (latlng[0]).toFixed(2) // show latitude of clicked point in the console
+          lon = (latlng[1]).toFixed(2) // show longitude of clicked point in the console
+          
+        // 3- Add coordinates of the selected geometry in the div element 
+        //  with id = "box" 
+           document.getElementById("lat").innerHTML = lat;
+           document.getElementById("lon").innerHTML = lon;
+        });
+
+      /* Set custom infowindow (= click infowindow) More information here:
+      http://docs.cartodb.com/cartodb-platform/cartodb-js/ui-functions/#cartodbvisvisaddinfowindowmap-layer-fields--options
+
+      */
+
+       cdb.vis.Vis.addInfowindow(
+          map, layer, ['adm0name','name','pop_max'],
+          {
+             infowindowTemplate: $('#infowindow_template').html()
+          });
+
+       /* Set custom tooltip (= hover infowindow) More information here:
+            http://docs.cartodb.com/cartodb-platform/cartodb-js/ui-functions/#visaddoverlaytooltip
+
+         */
+
+        // 1- Define tootltip with createLayer
+         var tooltip = layer.leafletMap.viz.addOverlay({
+            type: 'tooltip',
+            layer: layer,
+            template: '<div class="cartodb-tooltip-content-wrapper"><p>{{name}}</p></div>', 
+            width: 200,
+            position: 'bottom|right',
+            fields: [{ name: 'name' }]
+          });
+
+          //2- Add tooltip to map object
+          $('body').append(tooltip.render().el);
+
+         /* Set custom infobox (= hover infowindow) More information here:
+            http://docs.cartodb.com/cartodb-platform/cartodb-js/ui-functions/#visaddoverlayinfobox
+          */
+        
+      // 1- Define infobox with createLayer()        
+
+       var infoBox = layer.leafletMap.viz.addOverlay({
+          type: 'infobox',
+          layer: layer,
+          template: '<div class="cartodb-tooltip-content-wrapper"><h3>Maximum Population</h3><p>{{pop_max}}<span> hab</span></p></div>', 
+          width: 150,
+          position: 'top|right'
+        });
+
+       //2- Add infobox to map object
+        $('body').append(infoBox.render().el);     
+
+        });
+
+        /* Set custom legend */
+
+        //1- Define custom legend
+        var legend = new cdb.geo.ui.Legend({
```

Comment:
Heads up @oriolbx , the `cdb.geo.ui.` functions are not supported. We should try to avoid using them in examples. See [Docs issue#686](https://github.com/CartoDB/docs/issues/686).